React/Views/RCTViewKeyboardEvent.m

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RCTViewKeyboardEvent.h"
+#import <React/RCTAssert.h>
+
+@implementation RCTViewKeyboardEvent
+
++ (instancetype)keyDownEventWithReactTag:(NSNumber *)reactTag characters:(NSString*)characters modifier:(NSUInteger)modifier {
+  RCTViewKeyboardEvent *event = [[self alloc] initWithName:@"keyDown"
```

Comment:
Should this be a constant? Isn't it duplicated in RCTViewManager? Or is that semantically different but happens to be the same value?

Reply:
Based off other events it looks like they can be different